src/cryptonote_core/blockchain.cpp

```diff
@@ -5892,9 +5965,13 @@ void Blockchain::send_miner_notifications(uint64_t height, const crypto::hash &s
   std::vector<tx_block_template_backlog_entry> tx_backlog;
   m_tx_pool.get_block_template_backlog(tx_backlog);
 
+  crypto::ec_point fcmp_pp_tree_root{};
+  if (m_hardfork->get_current_version() >= HF_VERSION_FCMP_PLUS_PLUS)
```

Comment:
Same here, should be equal to `major_version`, yeah? And what we care about is the version of the aforementioend block anyways, yeah?

Reply:
Ya, I'm just going to use `major_version` here